<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerWindow</class>
 <widget class="QDialog" name="LayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="text_FC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zawartość frakcji drobnych (FC):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LayerWindow_button_OK">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LayerWindow_button_cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valField_layerThickness">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Jednostka: [m]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_layerThickness">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Miąższość warstwy:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valField_layerName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valField_FC">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Jednostka: [-]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_layerName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa warstwy:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
